pkg/server/helpers.go

```diff
@@ -95,10 +95,10 @@ const (
 )
 
 const (
-	// sandboxMetadataLabel is label name that identify metadata of sandbox in CreateContainerRequest
-	sandboxMetadataLabel = "io.cri-containerd.sandbox.metadata"
-	// sandboxMetadataLabel is label name that identify metadata of container in CreateContainerRequest
-	containerMetadataLabel = "io.cri-containerd.container.metadata"
+	// sandboxMetadataExtension is a extension name that identify metadata of sandbox in CreateContainerRequest
+	sandboxMetadataExtension = "io.cri-containerd.sandbox.metadata"
+	// containerMetadataExtension is a extension name that identify metadata of container in CreateContainerRequest
+	containerMetadataExtension = "io.cri-containerd.container.metadata"
```

Comment:
We still need a label to select containers and sandboxes.

Can we add a label `containerKindLabel = "io.cri-containerd.kind"`? And we should put "sandbox" on sandbox container, and "container" on application container.

Reply:
done

Reply:
What do you mean?   Should we use  containerKindLabel  but containerMetadataExtension?

Reply:
I mean instead of extension, we still need a label to indicate whether a container is sandbox container or regular container.

We could add a label `io.cri-containerd.kind` on each container, `io.cri-containerd.kind=sandbox` means it is a sandbox container, `io.cri-containerd.kind=container` means it is a regular container.

Reply:
Do you mean we shoule add both lable and extension?

Reply:
For example
```
opts := []containerd.NewContainerOpts{
		containerd.WithSnapshotter(c.config.ContainerdSnapshotter),
		containerd.WithNewSnapshot(id, image.Image),
		containerd.WithSpec(spec, specOpts...),
		containerd.WithContainerLabels(map[string]string{containerKindLabel: "sandbox"})
		containerd.WithContainerExtension(sandboxMetadataExtension, &sandbox.Metadata),
		containerd.WithRuntime(defaultRuntime, nil)}
	container, err := c.client.NewContainer(ctx, id, opts...)
```